src/components/checkout.summarylist.less

```diff
@@ -24,6 +24,12 @@
 .cart-summary-list {
   padding: 0;
   
+  label {
```

Comment:
Prefer a class name to an element name. With entire element styled if anyone ever adds another `label` to this component it'll get styled automatically.